src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -240,6 +237,11 @@ function redirectToPool(gauge: VotingGaugeWithVotes) {
           />
         </div>
       </template>
+      <template #nextPeriodVotesCell="gauge">
+        <div v-if="!isLoading" class="px-6 py-4">
```

Comment:
I'm not sure why the `align` property isn't doing this but please fix the alignment:
```suggestion
        <div v-if="!isLoading" class="px-6 py-4 text-right">
```

Reply:
This makes the text in the popup alight along the right hand side. Why did you want this changed?

Reply:
It looks like this:
![tooltip](https://user-images.githubusercontent.com/525534/159635732-12c00219-c7fa-487e-8a50-ad15fe9b7c19.png)